paasta_tools/readiness_check.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import logging
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description=(
+            'paasta_readiness_check is a tool to verfiy if changes are safe to '
+            'be made to a host.\n\n'
+            'It does this by inspecting the running tasks on localhost, and returns '
+            'non-0 if any of thoses tasks are "at-risk"\n\n'
+            'A task is at-risk if it is low in replication as determined by the replication '
```

Comment:
We were also using this terminology to refer to tasks running on a host marked as `draining` in `paasta_maintenance`. This isn't the first overloaded term we've used in PaaSTA. Maybe we should re-evaluate our terminology across the board and attempt to clear some of this up.